src/main/java/ch/bfh/bti7081/view/MainViewContent.java

```diff
@@ -0,0 +1,31 @@
+package ch.bfh.bti7081.view;
+
+import com.vaadin.flow.component.html.H1;
+import com.vaadin.flow.component.html.H2;
+import com.vaadin.flow.component.html.Label;
+import com.vaadin.flow.component.orderedlayout.HorizontalLayout;
+import com.vaadin.flow.component.orderedlayout.VerticalLayout;
+
+public class MainViewContent extends VerticalLayout {
+    private String loremipsum = "Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo dolores et ea rebum. Stet clita kasd gubergren, no sea takimata sanctus est Lorem ipsum dolor sit amet.";
+
+    public MainViewContent() {
+        // title and welcometext
+        this.add(new H1("Herzlich Willkommen!"));
+        this.add(new Label(loremipsum));
+    }
+
+    public void setFeatureView(VerticalLayout seminariesLayout) {
```

Comment:
Hier könnte z.B. auch eine Liste der Daten übermittelt werden, die durch einen Loop in die Vaadin-Objekte verschachtelt wird.

Reply:
gute Idee, wird angepasst